policy/rethinkdb/rethinkdb.go

```diff
@@ -0,0 +1,240 @@
+package rethinkdb
```

Comment:
maybe add a package description like:

`package rethinkdb is a ladon storage backend for rethinkDB.`